libraries/oneview_helper.rb

```diff
@@ -55,10 +60,12 @@ def self.load_sdk(context)
     rescue LoadError => e
       Chef::Log.debug("Could not load gem oneview-sdk #{node['oneview']['ruby_sdk_version']}. Message: #{e.message}")
       Chef::Log.info("Could not load gem oneview-sdk #{node['oneview']['ruby_sdk_version']}. Making sure it's installed...")
+      # :nocov:
```

Comment:
I don't think it does get executed by default because in rspec, it basically runs in --why-run mode, meaning only the resources that we say it should step into get executed. This is why we do [this](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/spec_helper.rb#L58). I tested it out by putting a `puts` statement inside this block and running the tests, and it never printed anything. And I think it's OK that it doesn't execute that. I think there's even a test somewhere that mocks it out.